components/camel-kafka/src/main/docs/kafka.adoc

```diff
@@ -0,0 +1,307 @@
+[[Kafka-KafkaComponent]]
+Kafka Component
+~~~~~~~~~~~~~~~
+
+*Available as of Camel 2.13*
+
+The *kafka:* component is used for communicating with
+http://kafka.apache.org/[Apache Kafka] message broker.
+
+Maven users will need to add the following dependency to their `pom.xml`
+for this component.
+
+[source,xml]
+------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-kafka</artifactId>
+    <version>x.x.x</version>
+    <!-- use the same version as your Camel core version -->
+</dependency>
+------------------------------------------------------------
+
+[[Kafka-Camel2.17ornewer]]
+Camel 2.17 or newer
++++++++++++++++++++
+
+Scala is no longer used, as we use the kafka java client.
+
+[[Kafka-Camel2.16orolder]]
+Camel 2.16 or older
++++++++++++++++++++
+
+And then the Scala libraries of choice. camel-kafka does not include
+that dependency, but assumes it is provided. For example to use Scala
+2.10.4 add:
+
+[source,xml]
+--------------------------------------------
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>2.10.4</version>
+    </dependency>
+--------------------------------------------
+
+[[Kafka-URIformat]]
+URI format
+^^^^^^^^^^
+
+[source,java]
+---------------------------
+kafka:server:port[?options]
+---------------------------
+
+ 
+
+[[Kafka-Options]]
+Options (Camel 2.16 or older)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+
+
```

Comment:
What's this file, we have an auto-generated file in src/main/docs called kafka-component.adoc, all the manual changes to documentation must be there. Can you remove this?